server/controllers/postController.ts

```diff
@@ -1,12 +1,13 @@
 import { initManagers } from '../db/initDB';
+import { SortOption } from '../db/dbManager';
 
 const { postManager, userManager } = initManagers();
 
 async function postPosts(req, res, next) {
 	const refreshToken = req.cookies.refreshToken;
 	const username = await userManager.findRefreshToken(refreshToken);
 	let result;
-	const { body: { title }, body: { postBody } } = req;
+	const { body: { title }, body: { body: postBody } } = req;
 	if (req.body.parent) {
 		const { body: { parent } } = req;
```

Comment:
You use destructurization several times. Make it once and in the most simplest way.